code/game/objects/items/stunbaton.dm

```diff
@@ -15,24 +15,22 @@
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
 
 	throwforce = 7
-	var/throw_stun_chance = 35
-
+	//The likelihood of a thrown baton to apply it's stun effect. Otherwise, it's simply a standard non-stun hit.
+	var/throw_stun_chance = 30

```

Comment:
📝

Reply:
```suggestion
	///The likelihood of a thrown baton to apply it's stun effect. Otherwise, it's simply a standard non-stun hit.
	var/throw_stun_chance = 30

```